bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -248,8 +248,15 @@ spec:
             else
               # either we need to initialize a new cluster or wait for master to create it
               if [[ "${K8S_NODE_IP}" == "${CLUSTER_INITIATOR_IP}" ]]; then
+                # set DB election timer at DB creation time if OVN supports it
+                election_timer=
+                if test -n "$(/usr/share/ovn/scripts/ovn-ctl --help 2>&1 | grep "\--db-nb-election-timer")"; then
+                  election_timer="--db-nb-election-timer=$(({{.OVN_NB_RAFT_ELECTION_TIMER}}*1000))"
+                fi
+
                 exec /usr/share/ovn/scripts/ovn-ctl ${OVN_ARGS} \
                 --ovn-nb-log="-vconsole:${OVN_LOG_LEVEL} -vfile:off -vPATTERN:console:{{.OVN_LOG_PATTERN_CONSOLE}}" \
+                ${election_timer} \
```

Comment:
so after we set it here  on startup, what happens here: https://github.com/openshift/ovn-kubernetes/blob/c679b0c662b4386e4da9e6be78f4eeeed7157f40/go-controller/pkg/ovndbmanager/ovndbmanager.go#L111 and  https://github.com/openshift/ovn-kubernetes/blob/c679b0c662b4386e4da9e6be78f4eeeed7157f40/go-controller/pkg/ovndbmanager/ovndbmanager.go#L316? I am a but confused of what db checker is doing...

Reply:
oh Dan explains it https://bugzilla.redhat.com/show_bug.cgi?id=2041546#c0